src/foam/nanos/auth/User.js

```diff
@@ -158,6 +158,7 @@ foam.CLASS({
       containsPII: false,
       documentation: 'The username of the User.',
       section: 'userInformation',
+      required: true,
```

Comment:
this implies that Contact and Businesses need this required too.
How about a validationPredicate that defines required iff type= user

Reply:
will addresss.